mindmeld/augmentation.py

```diff
@@ -62,6 +76,7 @@ def create_augmentor(self):
             )
         try:
             # Validate configuration input
+            app_path = self.config.get("app_path")
```

Comment:
Clarification: .get() for "app_path" and "retain_entities" among other values are all in this try block. However, aside from "app_path" the other .get() invocations have alternate defaults. If this is the case, then the 'except KeyError as e' would only apply if the app_path was missing?

Reply:
It would apply either in the case or when an incorrect key is invoked for the AUGMENTATION MAP when a wrong augmentor class is provided (line 94)

Reply:
In that case we may want to move the other get() invocations before the try block to make this more clear